index.js

```diff
@@ -97,11 +98,11 @@ const manageRoutes = async () => {
         else if (route === "#assignkits") assignKitsScreen(auth, route);
         else if (route === "#kitsreceipt") kitsReceiptScreen(auth, route);
         else if (route === "#kitscsv") kitCsvScreen(auth, route);
-        else if (route === "#participantselection") printAddressesScreen(auth, route);
+        else if (route === "#kit_status_reports") kitStatusReportsShippedScreen(auth, route); // Temporarily make kitStatusReportsScreen route to kitStatusReportsShippedScreen route
```

Comment:
Should we stick to camelCase for style consistency?

Reply:
The majority of the routes are not camelCase but all lowercase. Should I maintain consistency with the majority and change `#kit_status_reports` to `kitstatusreports` ? Only a few routes are camelCase. I am okay with changing this route to camelCase.

Reply:
If you happen to change it, maybe change it to camelCase.